testhelpers/helm_helper.go

```diff
@@ -38,11 +47,39 @@ func RenderNode(node helm.Node, config interface{}) ([]byte, error) {
 	if err := helm.NewEncoder(&helmConfig).Encode(node); err != nil {
 		return nil, err
 	}
-	tmpl, err := template.New("").Parse(string(helmConfig.Bytes()))
+
+	// The functions added here are implementations of the helm
+	// functions used by fissile-generated templates.
+	//
+	// For quicker implementation of the tests we simply implement
+	// functions as needed, instead of going and importing sprig,
+	// etc. Less/No vendoring needed.
```

Comment:
Hmm, I'm just vendoring sprig in #349 because that will be easier when we use more of it. That and we don't have to review that code…

Reply:
I know, see my note about the conflict in the PR's description. Assuming that the tests land after merging 349 I will drop my copies of sprig functions and rework the setup to get the sprig function table and then extend it with the helm fakes before registering them all with the template.